websites/docs-smoke-test/src/data/navigation.yaml

```diff
@@ -57,3 +55,9 @@
     - title: BetaPageWithLongNameThatOverflowsBecauseItsOneWord
       path: /empty-pages/empty-page-four
       beta: true
+- chapter-title: Self-learning course
```

Comment:
Temporarily added to check how it behaves with courses where the user is not enrolled